packages/migration/src/migrations/v0036_changeBaseColorToHex.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function changeBaseColorToHex({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function getHexFromSolidArray(baseColor) {
```

Comment:
@swissspidy This function appears twice in the code base. Should it just live in the pattern package and migration imports it? Or the other way around?

Reply:
Migration should not import from another package, in order to keep it as lightweight as possible and prevent unexpected changes should the other package's implementation ever change.

So what you have now is OK.